src/tesk_core/filer.py

```diff
@@ -418,6 +420,148 @@ def ftp_make_dirs(ftp_connection, path):
         return 1
     return 0
 
+
+def get_path_folders(whole_path):
+    """
+    Returns all subfolders in a path, in order
+
+
+    >>> subfolders_in('this/is/a/path')
+    ['this', 'is', 'a', 'path']
+    """
+    path_fragments = whole_path.lstrip('/').split('/')
+    path = path_fragments[0]
+    subfolders = [path]
+    for fragment in path_fragments[1:]:
+        subfolders.append(fragment)
+    return subfolders
```

Comment:
This code is unnecessary. You assemble `subfolders` but already have `path_fragments` which you can return.